### ISSUE TITLE
`passwd` command not functioning

### ISSUE THREAD

**Issue by naortega** [pwman3/pwman3#49]
I'm trying to change the password on my database, but it seems as though it won't let me or this functionality is not implemented. When I attempt to run the command `passwd` I just get a new prompt:
```
pwman> passwd
pwman>
```
The password seems to have remained the same as well.

**Comment by oz123** [comment]
As part of the huge refactoring I did, the encryption mechanism was changed.
As such `passwd` was left to the end and forgotten.
It was not easy implement this feature with the new code base. The most easy work around for you would be simply to export the file to csv, and then re-import it to a new database, using then a new password. 

Implementing this feature without export would be to write a new table with the password,
and then removing the old table.
